pmd-java/src/main/java/net/sourceforge/pmd/lang/java/oom/ClassStats.java

```diff
@@ -159,14 +159,14 @@
      *
      * @return The result of the computation, or {@code Double.NaN} if it couldn't be performed
      */
-    /* default */ double computeWithResultOption(OperationMetricKey key, ASTAnyTypeDeclaration node,
+    /* default */ double computeWithResultOption(MetricKey<OperationMetric> key, ASTAnyTypeDeclaration node,
```

Comment:
Is the use of boilerplate `/* default */` comments for Java default scope (colloquially known as package scope) PMD style?

This is something I tend to see with newer Java programmers unfamiliar with the language default behaviors, similar to declaring `interface` methods `public` or `abstract`, etc.  I wouldn't think we would want it.  Alternatively, there are cases were descriptive comments indicating _why_ a class does not have broader scope are useful, but that's not this, and not sure it's warranted here.